_posts/platform/app/2000-01-01-tasks.md

```diff
@@ -70,12 +70,14 @@ writing something to _stdout_ or _stderr_.
 
 If the `--detached` option is set, the container will be started as a background
 one-off container. In this case the container is started instantly, logs from
-the job are aggregated to the total logs of the application. You have to make
-sure this job ends at some point.
+the job are aggregated to the total logs of the application. 
 
-{% warning %}
-A detached one-off cannot be killed. You must make sure the detached one-off does **not** execute indefinitely.
-{% endwarning %}
+You should make sure the job ends at some point. 
+If it went rogue, you can stop it with `--one-off-stop` option.
```

Comment:
There is no `--one-off-stop` option, but a `one-off-stop` command as you show in the example below.